src/schnetpack/interfaces/ase_interface.py

```diff
@@ -249,20 +249,28 @@ def __init__(
         # Container for basic ml model ouputs
         self.model_results = None
 
-    def _load_model(self, model_file: str) -> schnetpack.model.AtomisticModel:
+    def _load_model(
+        self, model_file: Union[str, schnetpack.model.AtomisticModel, torch.nn.Module]
+    ) -> Union[schnetpack.model.AtomisticModel, torch.nn.Module]:
         """
         Load an individual model, activate stress computation
 
         Args:
-            model_file (str): path to model.
+            model_file (None): Either path to model or model object
 
         Returns:
            AtomisticTask: loaded schnetpack model
         """
 
-        log.info("Loading model from {:s}".format(model_file))
-        # load model and keep it on CPU, device can be changed afterwards
-        model = load_model(model_file, device=torch.device("cpu")).to(torch.float64)
+        if isinstance(model_file, str):
+            log.info("Loading model from {:s}".format(model_file))
+            # load model and keep it on CPU, device can be changed afterwards
+            model = load_model(model_file, device=torch.device("cpu")).to(torch.float64)
```

Comment:
Why not use `self.device` instead of `"cpu"`?

Reply:
this is legacy. In the SpkCalculator the model is first loaded to the cpu and then moved to device. I am not sure why its done this way

Reply:
Ok, so we can just change it to `self.device`?

Reply:
I would say so